gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/Avatar.kt

```diff
@@ -77,6 +88,16 @@ public fun Avatar(
                 avatarQueryOptions = avatarQueryOptions,
             )
         }
+
+        UserProfileState.Empty -> Avatar(
+            model = if (isNightModeEnabled()) {
+                R.drawable.empty_profile_avatar_dark
+            } else {
+                R.drawable.empty_profile_avatar
+            },
```

Comment:
At first I tried using the folder `drawable-night` and it works well when you switch between light and dark modo in the device. However, we you override the `uiMode` as [we do in the demo app](https://github.com/Automattic/Gravatar-SDK-Android/blob/d43fab05636329659b124b49f525b739200f415c/demo-app/src/main/java/com/gravatar/demoapp/ui/DemoGravatarApp.kt#L172), it still shows the drawable for the device theme. 

For that reason I've written this workaround.